_ml-commons-plugin/model-access-control.md

```diff
@@ -0,0 +1,593 @@
+
+Users can be assigned one or more [_backend roles_]({{site.url}}{{site.baseurl}}/security/access-control/index/) to configure fine-grained access to models based on [_roles_]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index#permissions). A backend role helps map a set of users to the same role. For example, users with the `IT` backend role may be assigned the `ml_full_access` role that grants full access to all ML Commons features.
+
+## Model groups
+
+For access control, models are organized into _model groups_---collections of versions of a particular model. Like users, model groups can be assigned one or more backend roles. All versions of the same model share the same model name and have the same backend role or roles. 
+
+You are considered a model _owner_ when you create a new model group. You remain the owner of the model and all its versions even if another user registers a model to this model group. When a model owner creates a model group, the owner can specify one of the following _access modes_ for this model group:
+
+- `public`: All users who have access to the cluster can access this model group.
+- `private`: Only the model owner or an admin user can access this model group.
+- `restricted`: The owner, an admin user, or any user who shares one of the model group's backend roles can access any model in this model group. When creating a `restricted` model group, the owner must attach one or more of the owner's backend roles to the model. 
+
+An admin can access all model groups in the cluster regardless of their access mode.
+{: .note}
+
+## Model access control prerequisites
+
+To start using model access control, first perform the following steps:
+
+1. Enable the Security plugin on your cluster. For more information, see [Security in OpenSearch]({{site.url}}{{site.baseurl}}/security/). 
+2. For `restricted` model groups, ensure that an admin has [assigned backend roles to users](#assigning-backend-roles-to-users).
+3. [Enable model access control](#enabling-model-access-control) on your cluster. You can enable model access control dynamically by setting `plugins.ml_commons.model_access_control_enabled` to `true`.
+
+If either of the prerequisites is not met, all models in the cluster are `public` and can be accessed by any user who has access to the cluster.
+{: .note}
+
+## Assigning backend roles to users
+
+Create the appropriate backend roles and assign those roles to users. Backend roles usually come from an [LDAP server]({{site.url}}{{site.baseurl}}/security/configuration/ldap/) or [SAML provider]({{site.url}}{{site.baseurl}}/security/configuration/saml/), but if you use the internal user database, you can use the REST API to [add them manually]({{site.url}}{{site.baseurl}}/security/access-control/api#create-user).
+
+Only admin users can assign backend roles to users.
+{: .note}
+
+For example, consider two users: `alice` and `bob`.
```

Comment:
I would set up this example with different language. Something like, "When assigning backend roles, consider the following example:"

And I think the language from the Security Analytics Security page would fit better here, too (same example but worded better).
https://opensearch.org/docs/latest/security-analytics/security/